Change: value8 => valueC

@package vaimo/composer-patches-target1

@assert vaimo/composer-patches-target1,value8,value8

--- src/example.txt.org	2018-10-12 02:16:43.285930514 +0200 
+++ src/example.txt	2018-10-12 02:16:43.285930514 +0200 
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-value100
+valueC
 value9
 value10
 value11
